docs/src/main/paradox/config-checker.md

```diff
@@ -51,7 +51,6 @@ starts an actor system with the production like configuration. This test can be
 configured to fail the startup of the actor system when configuration issues are found.
 An *IllegalArgumentException* will then be thrown from `ActorSystem.apply/create`
 if there are any issues.
-:
```

Comment:
Is this `. :` commonly used? I could only find it in this page and on another one on akka main docs and was curious if it should stay. At first I thought it was a typo but then noticed the others as well..

Reply:
I don't know what that is. Seems unnecessary.